src/WebJobs.Script/Diagnostics/DiagnosticEventConstants.cs

```diff
@@ -10,5 +10,8 @@ internal static class DiagnosticEventConstants
 
         public const string ExternalStartupErrorCode = "AZFD0005";
         public const string ExternalStartupErrorHelpLink = "https://go.microsoft.com/fwlink/?linkid=2224847";
+
+        public const string SasTokenExpiringErrorCode = "AZFD0006";
+        public const string SasTokenExpiringErrorHelpLink = "https://go.microsoft.com/fwlink/?linkid=2241431"; // TODO: update this link with new documentation page
```

Comment:
Does this TODO need to happen after you merge the PR or before?

Reply:
It can happen either way! Fabio said we have a few weeks in between anyway when this change is merged in to update the documentation link, and as of right now it is pointing to documentation to switch to using identities (so it's still relevant)